### ISSUE TITLE
[Improvement] Custom checkbox

### ISSUE THREAD

**Issue by MichalNemec** [zitadel/zitadel#6228]
### Preflight Checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions
- [X] I have joined the [ZITADEL chat](https://zitadel.com/chat)

### Describe your problem

Make additional checkboxes for veryfing if user complies with text.
i.e.
[ ] I am 18+ years old.

### Describe your ideal solution

somewhere in settings to define these probably optional / required toggle.

### Version

Docker 81d82cbcbaefcc9be02c2c9a05ccee55e5eabc58f26c9eeb82d2c936cc9448f9

### Environment

ZITADEL Cloud

**Comment by hifabienne** [comment]
Hei @MichalNemec 
Do I understand you correctly that you want additional fields on the user registration page?
Our registration page, should cover the standard use cases. 
If you need more customization we recommend building your own registration page. 
You can add additional fields and store that values to the user metadata (key value).

**Comment by MichalNemec** [comment]
> Do I understand you correctly that you want additional fields on the user registration page?

Yes, I agree, that it should cover standard use case. If there cannot be any additions, i understand.
It was just a thought to expand it more, so people dont have to create completely custom login/register because of checkboxes and compliances.
